src/rmarkdown.ts

```diff
@@ -4,6 +4,8 @@ import {
   Event, EventEmitter, Position, Range, TextDocument, TextEditorDecorationType, window
 } from 'vscode';
 import { runChunksInTerm } from './rTerminal';
+import * as vscode from 'vscode';
```

Comment:
Can you import `Selection` by importing from line 4?

Reply:
Yes. Sorry for mistake. Done in https://github.com/Ikuyadeu/vscode-R/pull/465/commits/1d418d8a3767c2367702933089239048c059ad7e